terraform/lambda.tf

```diff
@@ -28,10 +28,10 @@ resource "aws_iam_role_policy_attachment" "add_s3" {
 }
 
 resource "aws_lambda_function" "email" {
-    filename = "../function.zip"
+    filename = "../../generated/function.zip"
     runtime = "python3.8"
     role = aws_iam_role.email_lambda.arn
-    function_name = "bpm-email-lambda"
+    function_name = "${local.prefix}-ingest"
```

Comment:
Could use `${terraform.workspace}` instead of setting another `${local.prefix}`

Reply:
Replaced with ${WORKSPACE} in all occurrences of local.prefix

Reply:
Done